### ISSUE TITLE
Tests failing: pymatgen?

### ISSUE THREAD

**Issue by liamhuber** [pyiron/pyiron_contrib#1074]
The notebooks tests are failing. It looks to me like the latest version of pymatgen has an internal inconsistency. I'm not sure where it's coming into our dependency tree in an unbound way, but in #1073 you can see the full error message in a comment and I'm currently experimenting with an explicit pin on pymatgen.

Pinning it explicitly is an undesirable solution for me, since, afaik, we depend on it only indirectly. However, #1073 is holding up https://github.com/pyiron/pyiron_base/pull/1466, which, as discussed in the meeting today, is one of the steps holding up a new minor version of `pyiron_base`, so it's important to me that we get this resolved in some sort of reasonable time frame.

**Comment by liamhuber** [comment]
Yep, pinning pymatgen to the penultimate release did the trick

**Comment by liamhuber** [comment]
@pmrv, my stack trace started with `from pyiron_contrib.tinybase.lammps import LammpsStaticTask`, so I'm going to pass responsibility for this to you for the next step

**Comment by pmrv** [comment]
Tinybase uses it directly, I'll take a look tomorrow, but does look like an issue of pymatgen.  My bad on not adding it to the deps though.

**Comment by liamhuber** [comment]
Right, I guess I could read the full trace 🤦‍♂️ I see in the second step tinybase imports directly, so I agree adding it to the deps directly is the right thing to do. If I have a chance to get to it today I'll try to get it added systematically and leave you the PR to look at tomorrow
